### ISSUE TITLE
_root/index.html should include twitter/layout/paginator.html

### ISSUE THREAD

**Issue by coolaj86** [ruhoh/ruhoh.rb#183]
I find that I keep forgetting to edit both places each time. Perhaps DRY it up by default with some sort of `include` or `yield` and allow the user to still create separate layouts for the two if they desire.

**Comment by plusjade** [comment]
I agree, the paginator was a proof of concept that I finally got to work across all collections! The feature in question is the next step of course.
